src/main/java/com/dev/bankingservice/service/impl/TransactionServiceImpl.java

```diff
@@ -0,0 +1,60 @@
+package com.dev.bankingservice.service.impl;
+
+import com.dev.bankingservice.entity.Account;
+import com.dev.bankingservice.entity.Transaction;
+import com.dev.bankingservice.repository.AccountRepository;
+import com.dev.bankingservice.repository.TransactionRepository;
+import com.dev.bankingservice.service.TransactionService;
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.EntityNotFoundException;
+import lombok.AllArgsConstructor;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class TransactionServiceImpl implements TransactionService {
+    private final TransactionRepository transactionRepository;
+    private final AccountRepository accountRepository;
+
+    @Override
+    public List<Transaction> getAllByAccount(int page, int size, Account account) {
+        PageRequest pageable = PageRequest.of(page, size, Sort.by("date").descending());
+        return transactionRepository.getAllByAccount(account.getAccountNumber(), pageable);
+    }
+
+    @Override
+    public void transferFunds(String accountFrom, String accountTo, Double amount) {
```

Comment:
You can add `@Transactional` annotation for this method

Reply:
Added annotation